packages/cursorless-engine/src/core/Debug.ts

```diff
@@ -29,16 +30,21 @@ export class Debug {
         break;
       // Production mode. Enable based on user setting.
       case "production":
+        this.isProduction = true;
         this.evaluateSetting();
         this.disposableConfiguration =
           ide().configuration.onDidChangeConfiguration(this.evaluateSetting);
         break;
     }
   }
 
-  log(...args: any[]) {
+  log(message: string) {
     if (this.active) {
-      console.log(...args);
+      if (this.isProduction) {
+        ide().log(message);
+      } else {
+        console.log(message);
+      }
```

Comment:
why are we doing this here rather than defining the `log` function in `NormalizedIde` to use `console.log`?  Seems kind of vscode-specific that we need to use `console.log` vs output channel depending on how we're running

Reply:
I guess we could.